<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMAudioWidget</class>
 <widget class="QWidget" name="GMAudioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>32</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>32</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>32</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CGMAudioWidget</string>
  </property>
  <widget class="QLabel" name="audioTitleLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>135</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>135</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>135</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="artistNameLab">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>2</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BPMLab">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>2</y>
     <width>35</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CGMColorLabel" name="emotionLab">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>12</y>
     <width>10</width>
     <height>10</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CGMColorLabel</class>
   <extends>QLabel</extends>
   <header>GMColorLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
